### PR TITLE
Add Maildroppa

### PR DESCRIPTION
**Pull request by MarcusBiel**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -940,6 +940,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [mailcatcher.me](https://mailcatcher.me/) — Catches mail and serves it through a web interface.
   * [Mailcheck.ai](https://www.mailcheck.ai/) - Prevent users to sign up with temporary email addresses, 120 requests/hour (~86,400 per month)
   * [Mailchimp](https://mailchimp.com/) — 500 subscribers and 1,000 emails/month free.
+  * [Maildroppa](https://maildroppa.com) - Up to 500 subscribers and unlimited emails as well as automations for free.
```
Comment (ripienaar) [inline README.md:943]: Look like less than 500 subs?

Screenshot 2024-04-16 at 12 14 08

**Review by ripienaar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MarcusBiel) [reply README.md:943]: True ;). Sorry I am not awake yet :)

Comment (ripienaar) [reply README.md:943]: > True ;). Sorry I am not awake yet :)

I can tell! you fixed the wrong entry :P

Comment (MarcusBiel) [reply README.md]: Ups. How embarassing. ADHD in action :D

**Comment by ripienaar** [comment]
Thanks :)

**Pull request merged**